provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/AdditionalData.java

```diff
@@ -18,4 +18,12 @@
     @Getter
     @Setter
     private String provisioningPayload;
+
+    /**
+     * The PEM encoded operational client certificate request that DPS will send to a linked certificate authority which
+     * signs and returns an X.509 certificate that the IoT device can use to authenticate itself with an IoT Hub.
+     */
+    @Getter
+    @Setter
+    public String operationalCertificateRequest;
```

Comment:
The choice of variable name on the .NET SDK (we'd like to keep the variable name same across all languages so as to keep the user experience similar): https://github.com/Azure/azure-iot-sdk-csharp/blob/c3008205c68de5b8fede085533048987e15af1c9/provisioning/device/src/ProvisioningRegistrationAdditionalData.cs#L23